src/schema/artwork/index.ts

```diff
@@ -503,6 +504,13 @@ export const artworkFields = () => {
       deprecationReason: "Prefer dimensions instead.",
     },
     meta: Meta,
+    pageviews: {
+      type: GraphQLInt,
+      description: "Weekly pageview data (static).",
+      deprecationReason:
+        "Do not use! This is for an AB test and will be imminently deprecated.",
+      resolve: ({ _id }) => artworkPageviews[`${_id}`],
```

Comment:
Why the need for string interpolation? Can’t this just be `artworkPageviews[_id]`?

Reply:
Yup, you're right.